### PR TITLE
[tune] Improve PBT example

### PR DESCRIPTION
**Pull request by ericl**

### CODE REVIEW

File: python/ray/tune/examples/pbt_example.py [hunk 1]
```diff
@@ -64,35 +90,29 @@ def reset_config(self, new_config):
...
         ray.init()
 
     pbt = PopulationBasedTraining(
         time_attr="training_iteration",
-        reward_attr="episode_reward_mean",
-        perturbation_interval=10,
+        reward_attr="mean_accuracy",
+        perturbation_interval=20,
         hyperparam_mutations={
-            # Allow for scaling-based perturbations, with a uniform backing
```
Comment (richardliaw) [inline python/ray/tune/examples/pbt_example.py]: Hm, there should be at least one example that showcases the many features of PBT. Maybe this triangle LR example should be a separate file?

Comment (ericl) [reply python/ray/tune/examples/pbt_example.py]: Hm I actually think it's better to have one good example. As a compromise, I added factor_2 back here. What do you think?

Comment (richardliaw) [reply python/ray/tune/examples/pbt_example.py]: I think this is fine for now; I think a better fix later is to have a very comprehensive walkthrough page in the Ray Examples section on how exactly to use PBT.

File: python/ray/tune/examples/pbt_example.py [hunk 2]
```diff
@@ -64,35 +90,33 @@ def reset_config(self, new_config):
...
-                "training_iteration": 20 if args.smoke_test else 99999
+                "training_iteration": 2000,
             },
-            "num_samples": 10,
+            "num_samples": 4,
             "config": {
-                "factor_1": 4.0,
-                "factor_2": 1.0,
+                "lr": 0.0001,
+                "some_other_factor": 1,  # note: has no effect
```
Comment (richardliaw) [inline python/ray/tune/examples/pbt_example.py]:
```suggestion
                "some_other_factor": 1,  # note: This parameter is not used in this example
```

**Review by richardliaw — APPROVED** [review]

**Pull request merged**